tests/Microsoft.Fx.Portability.Tests/Microsoft.Fx.Portability.Tests.csproj

```diff
@@ -33,6 +33,8 @@
   <ItemGroup>
     <EmbeddedResource Include="TestAssets\005- ListT.ForEach.md" />
     <EmbeddedResource Include="TestAssets\006- System.Uri.md" />
+    <EmbeddedResource Include="TestAssets\CategoryWithSpaces.md" />
```

Comment:
It may make it simpler to just replace all these with `<EmbeddedResource Include="TestAssets\*.md" />` or even just a `TestAssets\**\*` glob